runbot_travis2docker/models/runbot_branch.py

```diff
@@ -33,6 +34,29 @@ def _compute_name_weblate(self):
                         dict(match.groupdict(), branch=branch['branch_name']))
             branch.name_weblate = name
 
+    @tools.ormcache('ssh')
+    def _ssh_keyscan(self, ssh):
+        """This function execute the command 'ssh-keysan' to avoid the question
+        when the command git fetch is excecuted.
+        The question is like to:
+            'Are you sure you want to continue connecting (yes/no)?'"""
+        cmd = ['ssh-keyscan', '-p']
+        match = re.search(r'@(?P<host>[^/]+(?=:|/]))?(?::)?(?P<port>[^/]+)?/',
+                          ssh)
+        if not match:
+            return False
+        cmd.extend(sorted(match.groups('22')))
+        with open(os.path.expanduser('~/.ssh/known_hosts'), 'a+') as hosts:
+            new_keys = subprocess.Popen(cmd, stdout=subprocess.PIPE,
+                                        stderr=subprocess.PIPE
+                                        ).stdout.readlines()
```

Comment:
Avoid use `readlines` in order to save memory ram.
You can use
`for line in stdout:`

Reply:
Good point, to optimize the memory use